util/utils.go

```diff
@@ -4,12 +4,20 @@ import (
 	"math/rand"
 	"os"
 	"time"
+
+	"github.com/deepfence/vessel"
+	vesselConstants "github.com/deepfence/vessel/constants"
+	containerdRuntime "github.com/deepfence/vessel/containerd"
+	log "github.com/sirupsen/logrus"
 )
 
 const (
-	charset = "abcdefghijklmnopqrstuvwxyz" + "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+	charset        = "abcdefghijklmnopqrstuvwxyz" + "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+	containerdSock = "unix:///run/containerd/containerd.sock"
```

Comment:
Can this be made configurable using env?

Reply:
we can read from the env, we need to add env vars to docker-compose.yml and helm charts. Also we can keep this, in case if no env var is provided, default would fall back here.
would that make sense?